### PR TITLE
Fix ExpressJS for recent versions of chrome

### PR DESCRIPTION
**Pull request by alexanderGugel**
See http://stackoverflow.com/a/8658281/3773544 for further reference

### CODE REVIEW

File: server_expressjs.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 			</p>
 			<pre class="code">app.all('*', function(req, res, next) {
   res.header("Access-Control-Allow-Origin", "*");
-  res.header("Access-Control-Allow-Headers", "X-Requested-With");
+  res.header("Access-Control-Allow-Headers", "Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept");
```
Comment (monsur) [inline server_expressjs.html]: "Access-Control-Allow-Headers" exists twice, once in the key and once in the value. I think you want to remove it from the value.

**Comment by alexanderGugel** [comment]
Thanks, you're right. Has been removed now.

File: server_expressjs.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
 			</p>
 			<pre class="code">app.all('*', function(req, res, next) {
   res.header("Access-Control-Allow-Origin", "*");
-  res.header("Access-Control-Allow-Headers", "X-Requested-With");
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
```
Comment (monsur) [inline server_expressjs.html:15]: Looks good, thanks! Just curious, does this still work without "Origin" in the value? I can't imagine browsers rejecting requests if Origin isn't present (of course, it doesn't hurt to have it there either).

Comment (alexanderGugel) [inline server_expressjs.html:15]: Yeah, I agree. It doesn't make sense at all.

I tried in Chrome Edge and it doesn't work without.

Can you provide some details on the browser you tested it on?

Comment (monsur) [inline server_expressjs.html:15]: I was using test-cors.org to recreate a request. Here's an example; there's
no Origin in the Access-Control-Allow-Headers response (or in the
Access-Control-Request-Headers request either):

http://client.cors-api.appspot.com/client#?client_method=GET&client_credentials=false&client_headers=Foo%3A%20bar&server_enable=true&server_status=200&server_credentials=false&server_headers=Foo&server_tabs=local

Comment (alexanderGugel) [inline server_expressjs.html:15]: By the way: The bower registry does it also this way: https://github.com/bower/registry/blob/master/lib/cors.js#L4

**Comment by monsur** [comment]
I accepted your patch, but now I'm curious to get to the bottom of this issue. What browser and version are you using? Maybe certain browsers expect the Origin header?

**Comment by alexanderGugel** [comment]
I believe only the most recent versions of chrome.
I just tested it with Chrome 37.0.2062.124 and it only works with my commit.
Firefox 31 works without.

Testing it on Sauce Labs would be a good idea, probably going to spend some time on the weekend figuring out which browsers are affected.

**Pull request merged**